modules/plot/main/coffee/Sparkline.coffee

```diff
@@ -37,7 +37,7 @@ class Sparkline
     hx.components.clear(selector)
     hx.component.register(selector, this)
 
-    graph = new hx.Graph(selector)
+    graph = new hx.Graph(selector, options)
```

Comment:
The idea behind sparkline is that it is a graph for a specific purpose with _intentionally_ limited options that prevent you from making it non-sparkline. That being said, some of the options are useful so we could either allow all options to be set (and let people do what they want) or pass through only the options that wouldn't affect the sparkline-ness of the graph.
In it's current state, I wouldn't want to merge it as passing through the complete options object seems a bit haphazard and may introduce issues down the line as there are sparkline specific options being used